extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/ByteArrayTransfer.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.parquet.table.transfer;
+
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import org.jetbrains.annotations.NotNull;
+
+import java.nio.IntBuffer;
+
+final class ByteArrayTransfer extends PrimitiveArrayAndVectorTransfer<byte[], byte[], IntBuffer> {
```

Comment:
It looks to me like we should be adding these to `ReplicateParquetTransferObjects`.
You should replicate char to byte and short, and replicate int to long, float, and double.

Same logic applies for the vector codecs.